src/main/kotlin/io/github/cottonmc/jsonfactory/gens/Gens.kt

```diff
@@ -16,260 +16,260 @@ object Gens {
     /**
      * The `block/cube_all` (default) block model generator.
      */
-    val basicBlockModel: ContentGenerator = BasicBlockModel
+    val BASIC_BLOCK_MODEL: ContentGenerator = BasicBlockModel
```

Comment:
Renaming these properties is out-of-scope for this PR. Renaming these is something that I'm planning to do before the 0.5.0 release.

Reply:
It got caught in a refactor. I'll revert it.